plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php

```diff
@@ -64,7 +64,7 @@ public function get_description(): string {
 	/**
 	 * Adds the group blocks to the template.
 	 */
-	private function add_group_blocks() {
+	protected function add_group_blocks() {
```

Comment:
I thing this is required to be able to extend this template from a plugin for example. I know this is not part of this PR so it should not be a change here. Unless the plugin listed as part of the testing instruction already considers this change.

Reply:
@mdperez86 That's the reason. Otherwise, the sample plugin provided wouldn't work. It's a valid point whether it should be part of this PR or not. As I'm going to pursue creating a product template, I will probably close this PR.